qutebrowser/mainwindow/prompt.py

```diff
@@ -661,8 +675,11 @@ def _set_fileview_root(self, path, *, tabbed=False):
             log.prompt.exception("Failed to get directory information")
             return
 
-        root = self._file_model.setRootPath(path)
-        self._file_view.setRootIndex(root)
+        self._root_index = self._file_model.setRootPath(path)
```

Comment:
Please initialize this in `__init__`, i.e. `self._root_index = QModelIndex()` or so (this should also make mypy aware of what type it is).

Reply:
Ok done